modules/persistent-storage-local-tolerations.adoc

```diff
@@ -52,4 +52,17 @@ To configure local volumes for scheduling on tainted nodes:
 <4> The volume mode, either `Filesystem` or `Block`, defining the type of the local volumes.
 <5> The path containing a list of local storage devices to choose from.
 
+[NOTE]
+====
+To create local PVs on master nodes, modify the YAML file and add the `LocalVolume` spec, as shown in the following example:
```

Comment:
Should define "PVs" here, as it's the first time this acronym appears in this module.

Reply:
Addressed the change.